src/tcutility/analysis/vdd/charge.py

```diff
@@ -0,0 +1,18 @@
+﻿import attrs
+import numpy as np
+
+
+@attrs.define
+class VDDCharge:
```

Comment:
Is there a use-case for checking if two charges are equal?

Reply:
No, I removed the equal double underscore method for now